projects/github-actions/test-results-to-slack/README.md

```diff
@@ -58,6 +61,65 @@ The action relies on the following parameters.
 - (Required) `slack_channel` is the Slack channel ID where the messages will be sent to.
 - (Optional) `slack_username` is the Slack username the bot will use to send messages. Defaults to "GitHub Reporter".
 - (Optional) `slack_icon_emoji` is the icon emoji to use for messages. If not set it will use your app's default icon.
+- (Optional) `suite_name` is the name of the test suite. It will be included in the message, and it can also be used to define notification rules. See more in the Rules section.
+- (Optional) `rules_configuration_path` is the path to the configuration file that defines the rules. See more in the Rules section.
+
+### Rules
+
+You can configure different rules, to send notifications in multiple channels.
+
+There are two types of rules: refs rules, used to send notifications for specific branches or tags and suite rules, used to send notifications for specific test suites.
+
+#### Refs rules
+
+You can create as many rules as you want. For each rule, you need to define the ref type and ref name to match, and a list of channels to send the notification in case of a match. Optionally you can also define whether to exclude the default channel. By default, the default channel is not excluded and a notification will also be sent there.
+
+Example:
+
+```json
+{
+  "refs": [
+	{
+	  "type": "branch",
```

Comment:
What other ref types are available? Do you think it would make sense to mention them with a short description?

Reply:
I expect that if one is setting this up, there's already another build or tests workflow running for that refType, so they should already know what they have and need. 
Nevertheless, it wouldn't hurt mentioning something, so I added a small example with [8ad7721](https://github.com/Automattic/jetpack/pull/25830/commits/8ad772199a2a258f759be5fe882ee7dff732e23e)